openedx/features/offline_mode/utils.py

```diff
@@ -18,85 +19,95 @@
 log = logging.getLogger(__name__)
 
 
-def generate_offline_content(xblock, html_data):
+class SaveOfflineContentToStorage:
```

Comment:
This class name should be changed to be in OOP paradigm. The name is stating and action, and suitable for function, but not for class.

Also the class should be moved outside of utils file into a file similar to class name

Reply:
Yes, you are right. If you don't mind, I'll take that class name: `OfflineContentStorageManager` & storage_management.py file

Reply:
I think it would be better to name the class something like `OfflineContentGenerator` instead of renaming the main method to `generate`.